storage/client/queries/queries.go

```diff
@@ -401,6 +410,18 @@ const (
 			DECODE(evs.body ->> 'address', 'base64')=preimages.address_data AND
 			preimages.context_identifier = 'oasis-runtime-sdk/address: secp256k1eth' AND
 			preimages.context_version = 0
+		LEFT JOIN chain.address_preimages AS pre_from ON
+			evs.body->>'from' = pre_from.address AND
+			pre_from.context_identifier = 'oasis-runtime-sdk/address: secp256k1eth' AND
+			pre_from.context_version = 0
```

Comment:
don't need to filter these in the db, EthChecksumAddrFromPreimage does the checks internally